lib/src/err/mod.rs

```diff
@@ -78,8 +79,8 @@ pub enum Error {
 	TxConditionNotMet,
 
 	/// The key being inserted in the transaction already exists
-	#[error("The key being inserted already exists")]
-	TxKeyAlreadyExists,
+	#[error("The key being inserted already exists: {0}")]
+	TxKeyAlreadyExists(KeyCategory),
```

Comment:
@phughk This breaking change made it to main. We need to fix this.

Reply:
Created https://github.com/surrealdb/surrealdb/issues/2822 and will pick it up soon thanks @rushmorem !